docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+- Push a commit that updates the manifests for the different WGs, based on the _git revision_ they had provided.
+- Create a new **vX.Y-branch** branch in the manifests repo
+- Create a new **vX.Y-rc.1** tag that will be pointing to the branch created above
+- Add commits on top of the **vX.Y-branch** branch to include bug fixes. The other WGs can provide a new _git revision_ to Manifests WG leads. The new git revision must only include bug fixes, not new features.
+- On the last day of feature freeze, cut a new **vX.Y-rc.2** RC tag on the **vX.Y-branch** release branch.
+
+**Actions for the Release Team:**
+- Generate a changelog/feature summary for the community, to be presented at the community meeting
+    - This will allow WGs to highlight their work, and also act as a checkpoint for everyone to know that they can move to documentation and testing for the rest of the cycle.
+- Identify, early in the first week, features at risk. These should either be aggressively fixed or punted
+
+**Success Criteria:** All working group git branches created, manifests complete, features either have landed or been pushed to next release.
+
+### Manifests testing (1 week)
+
+The Manifests WG leads will be evaluating the final structure of manifests before Distribution owners start testing
+
+**Actions for other WGs:**
+- Provide updates and fixes for pending issues
+
+**Actions for Manifests WG:**
+- Ensure the manifests provide a deployable Kubeflow installation
+- Sync with other WGs on pending issues
+- After finished testing, create a new **vX.Y-rc.3** tag pointing at the commit of the **vX.Y-branch** release branch in case more bug fixes were cherry-picked
+
+**Success Criteria:** Manifests complete and fixes from working groups applied.
+
+### Documentation (3 weeks) (Starts the same time as Feature Freeze)
+
+Now that it’s been 3 weeks of fFeature freeze and continuous bug fixes we
+should all focus on ensuring that our docs are up to date, before distributions
+start testing. Features that are at risk or being worked up until the deadline
+can be documented last, ideally most of the feature work has landed already and
+those can start to be documented.
+
+**Actions for ALL WGs:** Help on keeping the docs up to date.
+
+**Actions for the Release Team:**
+- Request a list of features, from the Working Groups, that require updates to the documentation
+- Ensure the provided components versions match the documentation
+- Work alongside the Working Groups to bring the docs up to date
+
+**Success Criteria:** Documentation for this release completed. A new version
+in the website is cut.
+
+### Distribution testing (3 weeks)
+
+Distribution owners will start testing the latest RC tag and report back issues they find. Depending on the severity of the issues, and number of bug fixes that will accommodate them, the Manifests WG leads might need to cut a new RC tag.
```

Comment:
I think it's OK to omit it in this case since the final tag will be cut for the release based on the latest state of the release branch

Reply:
@kimwnasptd I assume the distribution will be testing with Git tags, and not off the release branch directly to avoid using possibly quickly changing manifests on the release branch. This way you can also more easily determine what changes might have introduced new issues.